src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -330,6 +330,7 @@ protected virtual bool IsAttributeConditionallyOmitted(NamedTypeSymbol attribute
                         }
 
                         BoundExpression boundNamedArgument = BindNamedAttributeArgument(argument, attributeType, diagnostics);
+                        this.Compilation.VariableUsePass.Visit(boundNamedArgument);
```

Comment:
>this.Compilation.VariableUsePass.Visit(boundNamedArgument); [](start = 24, length = 59)

Is this code getting hit on attributes used within local function declarations (on parameters, type parameters, etc.)?